res/css/views/rooms/_IRCLayout.pcss

```
@@ -81,7 +81,7 @@ $irc-line-height: $font-18px;
 
             > .mx_DisambiguatedProfile_displayName {
                 width: 100%;
-                text-align: end;
+                text-align: flex-end;
```

I guess this change is not necessary: https://developer.mozilla.org/en-US/docs/Web/CSS/text-align#syntax

---

Ah - thanks for catching that

```suggestion
                text-align: end;
```